docs/source/resources/software_stack.rst

```diff
@@ -0,0 +1,250 @@
+|                                                     | * 418.40 (or later R418)                                               |
+|                                                     | * 440.33 (or later R440)                                               |
+|                                                     | * 450.51 (or later R450)                                               |
+|                                                     | * 460.27 (or later R460)                                               |
+|                                                     |                                                                        |
+|                                                     | **NOTE**: The CUDA Driver Compatibility Package doesn’t support all    |
+|                                                     | drivers.                                                               | 
++-----------------------------------------------------+------------------------------------------------------------------------+
+| GPU Model                                           | * `NVIDIA Ampere GPU Architecture                                      |
+|                                                     |   <https://www.nvidia.com/en-us/data-center/ampere-architecture/>`_    |
+|                                                     | * `Turning <https://www.nvidia.com/en-us/geforce/turing/>`_            |
+|                                                     | * `Volta                                                               |
+|                                                     |   <https://www.nvidia.com/en-us/data-center/volta-gpu-architecture/>`_ |
+|                                                     | * `Pascal                                                              | 
+|                                                     |   <https://www.nvidia.com/en-us/data-center/pascal-gpu-architecture/>`_|
++-----------------------------------------------------+------------------------------------------------------------------------+
+| **Base Container Image**                                                                                                     |
++-----------------------------------------------------+------------------------------------------------------------------------+
+| Container Operating System                          | Ubuntu version 20.04                                                   | 
++-----------------------------------------------------+------------------------------------------------------------------------+
+| Base Container                                      | Triton version 21.06                                                   | 
++-----------------------------------------------------+------------------------------------------------------------------------+
+| CUDA                                                | 11.3                                                                   | 
```

Comment:
We've had a tough time getting CUDA 11.3 to work with cudf - so are skipping straight to 11.4 for the 0.6 release. 
```suggestion
| CUDA                                                | 11.4                                                                   | 
```